bundles/org.openhab.core.config.discovery.addon.ip/src/main/java/org/openhab/core/config/discovery/addon/ip/IpAddonFinder.java

```diff
@@ -197,6 +204,16 @@ private void scan() {
 
                 logger.trace("Checking candidate: {}", candidate.getUID());
 
+                // skip scanning if already installed
+                if (addonService != null) {
+                    @Nullable
```

Comment:
Is this annotation required?

Reply:
I removed it, SAT does not complain without.
To get rid of the last SAT warning I had to introduce a temporary copy of the AddonService :-/